build/tools/mingw/parameters.bat

```diff
@@ -0,0 +1,50 @@
+@echo off
+
+rem Number of jobs for parallels builds
+set JOBS=3
```

Comment:
This could be something like this :
```suggestion
if not "%NUMBER_OF_PROCESSORS%" == "" (
    set /A JOBS=%NUMBER_OF_PROCESSORS%-1
) else (
    set JOBS=3
)
```